src/common/error/error-response-body.ts

```diff
@@ -1,4 +1,12 @@
 export default interface ErrorResponseBody {
+    title?: string;
+    // todo: create different error response bodies, payment error responses use string
+    status?: number | string;
     type?: string;
-    errors?: Array<{ code: string }>;
+    detail?: string;
+    errors?: ErrorListResponseBody;
 }
+
+// todo: create different errorLists, payments use objects, order use strings
+export type ErrorListResponseBody =
+    Array<{ code: string; message?: string; } | null | string>;
```

Comment:
just to accomodate for a test, not sure if it's a real scenario but since there is a test for it...
<img width="587" alt="screen shot 2018-11-08 at 2 54 00 pm" src="https://user-images.githubusercontent.com/1621894/48176852-2c0cb600-e366-11e8-8241-79fb13945da6.png">